paymentsheet/src/test/java/com/stripe/android/paymentelement/confirmation/lpms/CashAppLpmTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.stripe.android.paymentelement.confirmation.lpms
+
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodCreateParams
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.ParameterizedRobolectricTestRunner
+
+@RunWith(ParameterizedRobolectricTestRunner::class)
```

Comment:
Should we add another parameterization type here? Maybe for billing details collection configuration? Anything else?